service/src/main/resources/quips/insultibot/insults.json

```diff
@@ -537,5 +504,214 @@
     "tags":[],
     "targetableText":"I talked to Santa Claus, and he says it's not within his power to bring [TARGET] a real friend for Christmas.",
     "targetableSsml":"I talked to Santa Claus, and he says it's not within his power to bring [TARGET] a real friend for Christmas."
+  },
+  {
+    "quipGroup":"QUITE_SIMPLE",
+    "text":"If someone were to ask me to describe you, it would be, quite simple.",
+    "ssml":"If someone were to ask me to describe you, it would be, quite simple.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":"If someone were to ask me to describe [TARGET], it would be, quite simple.",
+    "targetableSsml":"If someone were to ask me to describe [TARGET], it would be, quite simple."
+  },
+  {
+    "quipGroup":"ALPHABET_SOUP",
+    "text":"A bowl of alphabet soup has a larger vocabulary than you.",
+    "ssml":"A bowl of alphabet soup <break time=\"50ms\" /> has a larger vocabulary than you.",
```

Comment:
This is more of a problem with pacing due to us having things wrapped inside <s> tags. It rushes and distorts delivery. These super short breaks (which aren't really working) are more of stop-gap attempts to make it sound better in delivery.